sonic_platform_base/sonic_xcvr/api/public/cmisCDB.py

```diff
@@ -155,13 +155,13 @@ def query_cdb_status(self):
         '''
         This QUERY Status command may be used to retrieve the password acceptance
         status and to perform a test of the CDB interface.
-        It returns the reply message of this CDB command 0000h.
+        It returns the status and reply message of this CDB command 0000h.
         '''
         cmd = bytearray(b'\x00\x00\x00\x00\x02\x00\x00\x00\x00\x10')
         cmd[133-INIT_OFFSET] = self.cdb_chkcode(cmd)
         self.write_cdb(cmd)
         status = self.cdb1_chkstatus()
-        if (status != 0x1):
```

Comment:
@xinyulin Can you please revert this unrelated change to avoid increasing line history (agreed that the version without parenthesis is more pythonic though). Please do the same for similar changes in the diff.

Reply:
Sure, I have reverted it.